using-mobile-extensions/mobile-core/mobile-core-release-notes.md

```diff
@@ -1,5 +1,19 @@
 # Release Notes
 
+## March 12, 2020
+
+The following updates were made in this release:
+
+### iOS Core 2.5.1
```

Comment:
Same as above. You do not need to copy the Identity changes under the Core heading.